test/equippable.ts

```diff
@@ -132,24 +132,11 @@ async function assetsFixture() {
   return { equip, renderUtils };
 }
 
-async function multiAssetFixture() {
-  const equipFactory = await ethers.getContractFactory('RMRKEquippableMock');
-  const renderUtilsFactory = await ethers.getContractFactory('RMRKMultiAssetRenderUtils');
-
-  const equip = <RMRKEquippableMock>await equipFactory.deploy('equipWithEquippable', 'NWE');
-  await equip.deployed();
-
-  const renderUtils = <RMRKMultiAssetRenderUtils>await renderUtilsFactory.deploy();
-  await renderUtils.deployed();
-
-  return { equip, renderUtils };
-}
-
 // --------------- END FIXTURES -----------------------
 
 // --------------- EQUIPPABLE BEHAVIOR -----------------------
 
-describe('EquippableMock with Parts', async () => {
+describe('MinifiedEquippableMock with Parts', async () => {
```

Comment:
I think this shouldn't have been modified, right? Same for other instances in this file.. 🤔

Reply:
Actually yes. I use the name of the mock in the describe to identify it easily when all tests run

Reply:
Oh, I get it. Ok then 😁